spec/card_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+describe Card do
+  before do
+    @card = Card.create(original_text: "How are you?", translated_text: "Kamusta?", review_date: DateTime.new(2017, 2, 24))
+  end
```

Comment:
@davetoxa i haven't, but it seems that it's only best to use if i have a different variable to use in a method test. currently, all my test data comes from @card.